Helper/Data.php

```diff
@@ -192,12 +193,22 @@ public function getGeneralConfig($code, $storeId = null)
      */
     public function collectModuleConfig(): array
     {
-        $this->config['enabled'] = $this->scopeConfig->getValue('sentry/environment/enabled')
-            ?? $this->deploymentConfig->get('sentry') !== null;
-
-        foreach ($this->configKeys as $value) {
-            $this->config[$value] = $this->scopeConfig->getValue('sentry/environment/'.$value)
-                ?? $this->deploymentConfig->get('sentry/'.$value);
+        if (!isset($this->config['enabled'])) {
```

Comment:
Flipping this if check will help  reduce nested code blocks
```suggestion
        if (isset($this->config['enabled'])) {
            return $this->config;
        }
```

Reply:
I've applied this change in e4295a38f1a57d2c5b867078968517e4a58211e6.